### ISSUE TITLE
One-click "Remind Me" button on study page

### ISSUE THREAD

**Issue by LaurieBayet** [lookit/lookit-api#496]
**TL;DR**: A one-click button for people interested in participating in a study to receive a reminder at the date and email of their choice.

**Narrative**
"As an Experimenter on a study recruiting newborns, I would like to have a way for parents expecting a newborn child to very easily "sign-up" ahead of time, before their child is born, to receive a reminder about the study at a specific date in the near future (e.g. on the child's due date). the user would look at the study summary, click a "Remind Me" button, select a date of their choice on a pop-up calendar ("Remind me on..."), and enter their email or phone number ("Send a reminder at...") -- without leaving the study page. ". Clicking on the button triggers a reminder about the specific study at the date specified by the user. Ideally the user could opt to further "snooze" the reminder.

**Comment by kimberscott** [comment]
Additional options:
* Site-wide "sign up for a reminder" link for people who are pregnant or awaiting adoption placement
* Integration into registration form - so you can register an already-born child OR select some option to indicate the child isn't born/adopted yet, and get a reminder around the expected due date / placement. Would involve changing the Child model around a bit to accommodate this (no birthdate required, etc.) & not allowing participation from children until birthdate set. Also need to be thoughtful about what the reminder is like and what the user experience is for reporting miscarriage/stillbirth (including that we never accidentally remind you a second time to sign up).
